.gitignore

```diff
@@ -9,6 +9,9 @@ xcuserdata/
 *.xcscmblueprint
 *.xccheckout
 
+# Ignore Xcode scheme files generated by SwiftPM
+.swiftpm/xcode/xcshareddata/xcschemes/
```

Comment:
I was looking at some other `.gitignore` changes, and I noticed there's a whole SPM section commented out farther down - Maybe it makes the most sense to uncomment all of that and cleanup whats checked in today?

That could capture all of `.swiftpm` as well as quite a few other autogenerated things